Documentation/UserTsconfig/Setup.rst

```diff
@@ -98,7 +98,8 @@ edit_docModuleUpload
     boolean
 
 :aspect:`Description`
-    File upload directly in Doc. module
+    Allow file upload directly from within CE edit masks.
+    Caution: In most cases the uploaded file will in put into `fileadmin/user_upload/`.
```

Comment:
```suggestion
    Caution: In most cases the uploaded file will be stored in :file:`fileadmin/user_upload/`.
```

Maybe add an admonition? `.. important::` or `..note::` . Using caution admonition may be to harsh, I think.

Reply:
I reworked it and added cross-references, whoohoo ;-)